quickbooks/mixins.py

```diff
@@ -107,6 +109,7 @@ def send(self, qb=None, send_to=None):
         end_point = "{0}/{1}/send".format(self.qbo_object_name.lower(), self.Id)
 
         if send_to:
+            send_to = urllib.quote(send_to)
```

Comment:
This won't work in python 3. You have to use `urllib.parse.quote`.  This [stackoverflow](https://stackoverflow.com/questions/31827012/python-importing-urllib-quote) explains how to fix.

Reply:
You will also need to update `test_send_with_send_to_email ` test in `tests/unit/test_mixins.py`. You can click on the [TravisCI details](https://travis-ci.org/sidecars/python-quickbooks/builds/574483511?utm_source=github_status&utm_medium=notification) to see what tests are failing.

Reply:
@ej2 Had to use the 'future' module for both python 2 and 3 compatibility. Only the Python 3.3 test seems to be failing due to an unrelated issue.